### ISSUE TITLE
How to get annotation/highlighting features?

### ISSUE THREAD

**Issue by localjo** [foobnix/LibreraReader#716]
Thank you for your time an dedication working on this project. It's really great! I apologize if this question has been answered, but I searched and I am still unsure.

My question is about the annotation/highlighting feature that used to be available in this app. For me, this is a key feature of an ereader. I would like to know, what is the recommended way of getting it back?

I see according to this issue (https://github.com/foobnix/LibreraReader/issues/434) the feature was removed on the F-droid version after 8.1.191 because of a switch to MuPDF which doesn't support it well. There was a mention of fixing it in 2019.

I also see according to this issue (https://github.com/foobnix/LibreraReader/issues/617) that annotations only work for PDFs in scroll mode. I haven't been able to confirm this as all my books are in other formats.

And in the F-droid repo I see "For better security, the F-Droid version, is based on the latest MuPDF 1.16.1  ... for the above reason, some Internet-related features of Librera Reader are missing in its F-Droid version".

So my questions are:

- Is highlighting/annotation working in Librera PRO from Google Play Store? I do not use Google Play Store, but I would be happy to pay for this feature. Is there any other way to get Librera PRO?

- Is highlighting/annotation still working for PDFs in the F-Droid version? Is there still any plan to fix it for epub/mobi, etc?

- Is the only way to get highlight/annotation to downgrade to an F-Droid version before 8.1.191?

Thank you again for your work and any information you can share about the status of this feature!

**Comment by krillin666** [comment]
Any update on this :smile: ?

**Comment by ryanwwest** [comment]
I'd love this for epubs. Is OP implying that highlighting used to be an option for epub/mobi? I'd like them to just be saved directly to the file so other readers can view the highlights as well.

**Comment by azzamsa** [comment]
Any update about this?
This is the only missing feature that keep me using other app.

**Comment by EythorE** [comment]
Yes, please. More and better annotation tools woul be great. Currently I am annotating non-highlightable pdf by freehand drawing circles around or lines under text and adding a bookmark with notes. I'd like more tools to freehand draw on pdfs. For example adding text notes anywhere on the page and transparent colors (for highlighting). Bought the PRO version immediately after I saw the code was available here :).

**Comment by thebodzio** [comment]
Also it would be great if annotations worked in book mode. I fail to see any reason, from the usability point of view, why they should be restricted only for scroll mode.

My needs are very simple: highlight text and add a textual comment for it.

**Comment by Yolakalemowa** [comment]
This still not a thing?
